home_audit.node.js

```diff
@@ -1809,9 +1810,12 @@ function get_validation_messages (homeValues, requiredFields, additionalRules) {
 function validate_home_audit (homeValues, additionalRules = null) {
     // Pass homeValues into the scope of this file so that validation rules can reference it
     // without us having to explicitly pass it to every function
-    let requiredFields = require('./required_fields.node')(homeValues);
-
-    return get_validation_messages(homeValues, requiredFields, additionalRules);
+    let requiredFields = require('./required_fields.node.js')(homeValues);
+    if(homeValues.building) {
+        return requiredFields
```

Comment:
What is this check doing?

Reply:
Ahh, I see now... required_fields.node.js just became the entire holder for the new schema validation.

Reply:
Yup, and the nested version of the schema/handler just outputs the messages without needing to translate them via `get_validation_messages`